lib/api/1.1/southbound/profiles.js

```diff
@@ -31,9 +31,13 @@ function profilesRouterFactory (
      */
 
     router.get('/profiles', function (req, res) {
+        if (req.query.mac && req.query.ip) {
```

Comment:
@anhou  Are you looking for comments?  There are other routes missing query parameter annotations;  maybe there should be a story to track updating those as well.